update README (#46)
esqLABS/cts: README.Rmd

### Explore Compound

> Once you have imported a compound, you can explore its properties, such as formulations, protocols, and parameters:

Reviewer: ```suggestion
Once you have imported a compound model, you can explore its properties, such as formulations, protocols, and parameters:
```

Reviewer 2: I would replace "model" by "snapshot" in the sentence.

```{r, include=FALSE}
# Explore the formulations of Rifampicin
rifampicin$formulations
```

> This command outputs the formulation details of the compound, allowing you to understand how the compound is configured in the simulation.

Reviewer: ```suggestion
This command outputs the formulations that are defined for the compound and that can be used in simulations.
```
